src/Avalonia.Dialogs/AboutAvaloniaDialog.xaml.cs

```diff
@@ -7,12 +8,18 @@ namespace Avalonia.Dialogs
 {
     public class AboutAvaloniaDialog : Window
     {
+        private static readonly Version s_version = typeof(AboutAvaloniaDialog).Assembly.GetName().Version;
+
+        public static string Version { get; } = s_version.ToString(2);
```

Comment:
Maybe the third component is also worth mentioning?

Reply:
In development builds it's always 999, but in release builds it could make sense (although it's only patch and shouldn't matter that much).